motor_control/brushless/ramping.cpp

```diff
@@ -0,0 +1,65 @@
+#include "ramping.h"
+#include "GLOBAL.h"
+Ramping::Ramping():m_currentSpd(MIN_SPD), m_desiredSpd(MIN_SPD){}
+void Ramping::setCurrentSpd(byte spd)      {  m_currentSpd = spd; }
+void Ramping::setDesiredSpd(byte spd)      {  m_desiredSpd = spd; }
+byte Ramping::getCurrentSpd() const {  return m_currentSpd; }
+byte Ramping::getDesiredSpd() const {  return m_desiredSpd; }
+
+bool Ramping::isTimeToChangeSpd(long timeMillis){
```

Comment:
Why pass this parameter if its not used?

Reply:
forgot to change the internal function calls lol